src/dualscreen.tsx

```diff
@@ -75,6 +83,7 @@ export class DualScreen extends KalturaPlayer.core.BasePlugin implements IEngine
     ];
   }
   loadMedia(): void {
+    this._resetMode();
```

Comment:
need to reset the _readyPromise, (hopefully, the readiness mechanism works well in change media)